tools/Google.Cloud.Tools.ReleaseManager.IntegrationTests/ContainerCommands/BuildLibraryCommandTest.cs

```diff
@@ -121,7 +118,15 @@ public void PackageGroup()
             "--library-id=Google.Test");
         Assert.Equal(0, command.Execute(options));
 
-        codeRepo.AssertExist("build.sh.result.txt");
-        Assert.Equal(new string[] { "build-start", "--notests", "Google.Test.V1", "Google.Test.V2", "build-end" }, File.ReadAllLines(Path.Combine(codeRepo.Directory, "build.sh.result.txt")));
+        AssertScriptArguments(codeRepo, "--notests", "Google.Test.V1", "Google.Test.V2");
+    }
+
+    private void AssertScriptArguments(TestRepo repo, params string[] args) =>
+        AssertScriptOutput(repo, args.Prepend("build-start").Append("build-end").ToArray());
```

Comment:
Not blocking, but collection expressions are nice here:

```csharp
AssertScriptOutput(repo, ["build-start", .. args, "build-end"]);
```

(That's from memory, but I think it should work.)